docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Local Proxy to Remote Solana"
+proofedDate: 20230828
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: 
+---
+
+:::info
+The [Proxy local testing tutorial](/docs/operating/basic) describes how to perform testing with a local Proxy to a local Solana node. This option can be useful for developers who want to debug their Solidity contracts by hosting both Proxy and Solana nodes locally. This page provides a tutorial that demonstrates how to transact from a local Proxy to a public, remote Solana.
+:::
+
+
+## Prerequisites
+- [Docker](https://docs.docker.com/get-docker/); `docker-compose` v1.29 is recommended.
+- Operator keys. Test [public keys are available](https://github.com/neonlabsorg/neon-evm/tree/develop/evm_loader/program/config).
```

Comment:
@vodolaz Please confirm this is where someone having a play with local nEVM and local Solana pics up their Operator keys (where the pair is already embedded)